docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+2021-02-19T16:25:43.682+0000 (restore) recreated /var/opt/nuodb/archive/nuodb/demo; atoms=0
+2021-02-19T16:25:43.693+0000 curl -k  http://nginx.web.svc.cluster.local/20210219T160002.tar.gz | tar xzf - --strip-components 1 -C /var/opt/nuodb/archive/nuodb/20210219T160002-downloaded
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  100k  100  100k    0     0  9164k      0 --:--:-- --:--:-- --:--:-- 9164k
+2021-02-19T16:25:44.285+0000 restoring archive and/or clearing restored archive physical metadata
+2021-02-19T16:25:45.876+0000 Finished restoring /var/opt/nuodb/archive/nuodb/20210219T160002-downloaded to /var/opt/nuodb/archive/nuodb/demo. Created archive with archive ID 8
+2021-02-19T16:25:45.879+0000 removing /var/opt/nuodb/archive/nuodb/20210219T160002-downloaded
+2021-02-19T16:25:48.585+0000 Restore request for archiveId=1 marked as completed
+...
+```
+
+### Automatic archive initial import
+
+The _automatic archive initial import_ is a special case of a database restore operation.
+This operation allows the user to specify an external URL that contains a `tar.gz` archive copy or a backup set which will be automatically downloaded and used to define the database initial state, avoiding the need to do a post-deployment database load manually.
+Initial import is used for importing a backup taken from a different environment. It is configured in the `database.autoImport` section and will execute for every Storage Manager in the database deployment when all of the below conditions are met:
+
+- `database.autoImport.source` and `database.autoImport.type` are configured
+- the archive hasn't been initialized yet - `1.atm` is missing from the archive directory
+- no archive metadata is found for the archive in the admin layer or on disk
+
+To use automatic archive import in a pre-existing environment, it should be destroyed first so that the existing domain is ready to host the new database with the same name.
+
+1. Delete the database chart deployment.
+2. Remove the associated PVC, PV, and cloud storage volume that remains after database deletion.
+3. Remove the database from the domain by using `nuocmd delete database --db-name <db>`.
+4. List removed archives associated with the database by using `nuocmd show archives --db-name demo --removed`.
+5. Remove all archives associated with the database by using `nuocmd delete archive --archive-id <id> --purge`.
+
+The high-level steps to configure the automatic archive import operation to bootstrap a new database are the following:
+
+1. Prepare the import source and upload it to a remote location available to NuoDB Storage Manager pods.
+2. Set `database.autoImport` variables during NuoDB database chart installation.
+
+> **NOTE**: For simplicity _nginx_ deployment will be used in this example so that our remote source is available via HTTP in the Kubernetes cluster.
```

Comment:
_nginx_ is just some arbitrary name for a deployment? Should we choose something more generic sounding like _prod_, _qa_, or _dev_? _qa_ or _dev_ might make sense in this subsection, since that seems like a use-case for importing a database into a fresh domain.